exercises/practice/raindrops/.approaches/introduction.md

```diff
@@ -0,0 +1,51 @@
+# Introduction
+
+There are various ways to solve **raindrops**.
+Raindrops is a simple problem, but it can be solved in many different ways.
+Its focus is on checking for divisibility and building a string based on conditions.
+
+## General guidance
+
+To solve this exercise is some form of conditionals required.
+
+## Approach: Using conditionals
+
+Using conditionals to build a string using if expressions and the ternary operator.
+
+```crystal
+module Raindrops
+  def self.convert(count : Int32) : String
+    result = ""
+    result += "Pling" if count.divisible_by?(3)
+    result += "Plang" if count.divisible_by?(5)
+    result += "Plong" if count.divisible_by?(7)
+    result.blank? ? count.to_s : result
+  end
+end
+```
+
+For more information, check the [conditionals approach][approach-conditionals].
+
+## Approach: Using rules
```

Comment:
I'm not entirely sure about "rules", but I also can't really come up with a better alternative. I think the C approach that looks like this called it data-driven or something like that.